app/javascript/components/reconcilable.js

```diff
@@ -18,9 +19,16 @@ export default template({
   },
   methods: {
     searchForName: function () {
+      this.searchResourceType = 'person'
+      this.search(this.statement.person_name)
+    },
+    searchForParty: function () {
+      this.searchResourceType = 'party'
+      this.search(this.statement.parliamentary_group_name)
+    },
+    search: function (serachTerm) {
```

Comment:
For some reason my comment got lost on the wrong bit of code! This is the typo that needs fixing :)